src/main/kotlin/io/emeraldpay/dshackle/rpc/NativeCall.kt

```diff
@@ -481,17 +482,23 @@ open class NativeCall(
             ReaderData(ctx.upstream, ctx.upstreamFilter, ctx.callQuorum, signer, tracer),
         )
         val counter = reader.attempts()
+        var streamRequest = ctx.streamRequest
+        if (ctx.upstream.getChain().id in listOf(Chain.RIPPLE__MAINNET.id, Chain.RIPPLE__TESTNET.id)) {
```

Comment:
we can compere enums itself, not just id's

Reply:
Fixed
https://github.com/drpcorg/dshackle/pull/631/commits/def86003fff4c9e468f9f7b836d7aefee42ed567